build.gradle

```diff
@@ -22,7 +22,7 @@ ext {
     automatticTracksVersion = '3.3.0'
     gutenbergMobileVersion = 'v1.109.0'
     wordPressAztecVersion = 'v1.8.0'
-    wordPressFluxCVersion = '2.57.0'
+    wordPressFluxCVersion = '2911-115f8b738d8aa5c3cca37a5feae24aa7f0e399ff'
```

Comment:
@antonis @justtwago Just to remind, we should only merge a proper release or a `trunk-{commit_sha}` version of our dependencies to `trunk`. It's fine to use the `{pr_number}-{commit_sha}` dependencies for testing, but they should not be merged.

Reply:
Sorry for this @oguzkocer 😅 
I opened a small PR to update the reference https://github.com/wordpress-mobile/WordPress-Android/pull/19700

Reply:
No problem 👍  I am going to have to update it again while merging `release/23.8` to `trunk` which is how I noticed it in the first place 😅 So, I've approved the PR and merged it to avoid further conflicts. Hope that's OK!